python/ray/dashboard/utils.py

```diff
@@ -358,6 +358,31 @@ def address_tuple(address):
     return ip, int(port)
 
 
+def node_stats_to_dict(message):
```

Comment:
add type hints please

Reply:
added https://github.com/ray-project/ray/pull/51187/commits/75d4f5adda4ac4c9904aae9256992588297b36a5

Reply:
the type hint is derived from the return type of `node_stats`.

https://github.com/ray-project/ray/blob/2f8684f723b67d2d44070501292f345165835b3b/python/ray/_private/internal_api.py#L92.